users/data-scientist.md

```diff
@@ -0,0 +1,42 @@
+# User persona: Data Scientist
+
+_Fill out this template to define key aspects of a type of user the Operate First community wishes to address, i.e., the user persona._
+
+## Persona: User Type
+
+A data scientist can be considered as a special case, or sub-set, of a software engineer. They develop application that rely on the use of data sets, models and code. In the context of the Operate First community, this persona would primarily act as a user-developer; using the services provided by the OPF community cloud as well as hosting their own projects and services they have developed and will maintain on the community cloud.
```

Comment:
A few grammar and punctuation edits below. The main change is refactoring to have one sentence per line. This is the recommended practice for documentation, as it helps with reviews and easier-to-read diffs.

```suggestion
A data scientist can be considered as a special case, or sub-set, of a software engineer.
They develop applications that rely on the use of data sets, models, and code.
In the context of the Operate First community, this persona primarily acts as a user-developer; using the services provided by the Op1st community cloud as well as hosting their own projects and services they have developed and will maintain on the community cloud.
```

Reply:
Thanks @quaid . When you say. "one sentence per line" does that mean in the markdown file only? Or should it also render in the markdown as a list of sentences with spaces in between?

Reply:
I mean only in the markdown file